h/search/__init__.py

```diff
@@ -5,9 +5,11 @@
 from h.search.config import init
 from h.search.core import Search
 from h.search.core import FILTERS_KEY
+from h.search.query import TopLevelAnnotationsFilter
 
 __all__ = (
     'Search',
+    'TopLevelAnnotationsFilter',
```

Comment:
Adding `TopLevelAnnotationsFilter` to the official public API of `h.search`. In due time this will be done with other publicly-used filters and aggregations etc.